docs/DeveloperGuide.adoc

```diff
@@ -228,7 +249,7 @@ This section describes some noteworthy details on how certain features are imple
 
 // tag::undoredo[]
 === Undo/Redo feature
-The Undo and Redo feature are crucial to give users the flexibility of reverting wrongly executed commands.
+The Undo and Redo features are crucial to give users the flexibility of reverting wrongly executed commands.
```

Comment:
I thought Nazhou change it to "feature" in her PR?

Reply:
Reverting to original so Nazhou's PR can fix it.

```suggestion
The Undo and Redo feature are crucial to give users the flexibility of reverting wrongly executed commands.
```